.pre-commit-config.yaml

```
@@ -1,4 +1,4 @@
-# .pre-commit-config.yaml
+# .pre-commit-config.yaml 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)